### PR TITLE
ui: fix checkout for pending loans

### LINKED ISSUE #276: Assign ITEM to Pending loans
http://localhost:3000/backoffice/loans/18
we cannot assign an item to a pending loan.
we cannot checkout pending loan without assignment.
There is no way to checkout of the pending loan.
UX friendly solution:
Items which are not assigned should have "Assign and checkout" button to checkout in one click. This solution requires conditional rendering.

### PR DESCRIPTION
**Pull request by kpsherva**
* it gives a possibility to checkout pending loan using one
  of the items from the list

* minor fixes for loan actions

* available items list updated for the items not currently on loan

* ( closes #276 )

### CODE REVIEW

File: scripts/celery [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+celery worker -A invenio_app.celery -l INFO & pid_celery=$!
```
Comment (equadon) [inline scripts/celery:9]: :+1:

File: invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LoanDetails/components/AvailableItems/AvailableItems.js [hunk 2]
```diff
@@ -80,6 +105,15 @@ export default class AvailableItems extends Component {
     );
   }
 
+  _rowActionButton(row) {
+    const { loan } = this.props;
+    if (loan.metadata.state === 'PENDING') {
+      return this._checkoutItemButton(row, loan);
+    } else {
+      return this._assignItemButton(row);
```
Comment (ntarocco) [inline invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LoanDetails/components/AvailableItems/AvailableItems.js:113]: For me it is a bit unclear, in which case we assign the item? (I mean the else condition)

File: invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/UserDetails/components/ItemsSearch/ItemsSearch.js [hunk 3]
```diff
@@ -47,7 +47,10 @@ export default class ItemsSearch extends Component {
   _onPasteHandler = e => {
     let queryString = e.clipboardData.getData('Text');
     this.executeSearch(queryString).then(data => {
-      if (this.props.items.hits[0].status === 'LOANABLE') {
+      if (
+        this.props.items.hits[0].metadata.status === 'LOANABLE' &&
```
Comment (ntarocco) [inline invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/UserDetails/components/ItemsSearch/ItemsSearch.js]: `LOANABLE`?

Comment (kpsherva) [reply invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/UserDetails/components/ItemsSearch/ItemsSearch.js]: not rebased yet

File: invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LoanDetails/components/AvailableItems/AvailableItems.js (hunk 2 above)
Comment (kpsherva) [reply invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/LoanDetails/components/AvailableItems/AvailableItems.js:113]: we don't assign the item here, we render either the checkoutButton when the loan is `PENDING` or assign item button when the loan is any other state, to give the possibility to replace the item. I'm not sure how could I make it more descriptive, to leave a comment in here ?

**Review by ntarocco — DISMISSED** [review]

**Pull request merged**